File: LeetCode/1.)TwoSums/twoSums.java
package TwoSums;
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int arr[]={0,0};
        int length = nums.length;
        for(int i = 0; i< length; i++)
        {
            for(int j=i+1;j<length; j++)           
            {
                if (  nums[i]+nums[j] == target)
                {
                    arr[0]=i;
                    arr[1]=j;
                }
            }

        }
        return arr;
    }
}

File: LeetCode/1.)TwoSums/twoSums.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        # Brute force method of trying every possible combination from left to right.

        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                # check if i and (i+1),(i+2),... equals the target
                if nums[j] == target - nums[i]:
                    return [i, j]